source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -19,12 +19,12 @@ feedback: true
 Happy July, which means Home Assistant Core 2021.7!
 
 An interesting release, with a bunch of little goodies to make things easier in
-creating automations, scripts and doing templating. Those are things that in
+creating automations, scripts, and doing templating. Those are things that in
```

Comment:
We avoid using the oxford commas in all our communication and documentation.
This is documented in our style guides.

```suggestion
creating automations, scripts and doing templating. Those are things that in
```